libkbfs/folder_block_ops.go

```diff
@@ -986,8 +986,13 @@ func (fbo *folderBlockOps) RemoveDirEntryInCache(lState *lockState, dir path,
 	fbo.blockLock.Lock(lState)
 	defer fbo.blockLock.Unlock(lState)
 	undoFn := fbo.removeDirEntryInCacheLocked(lState, dir, oldName, oldDe)
+	unlinkUndoFn := fbo.nodeCache.Unlink(
+		oldDe.Ref(), dir.ChildPath(oldName, oldDe.BlockPointer), oldDe)
 	return fbo.wrapWithBlockLock(func() {
 		undoFn()
+		if unlinkUndoFn != nil {
```

Comment:
You're right, I fixed that up as per @jzila's comments in #951.  Will be fixed here next time I push.